Components/Card/card.css

```diff
@@ -0,0 +1,116 @@
+  .element-display {
+      display: flex;
+      justify-content: center;
+      flex-direction: row;
+      flex-wrap: wrap;
+      align-items: center;
+      padding: 1rem;
+      border-radius: 10px;
+      gap:15px;
+      margin-top: 1rem;
+  }
+    
+  .card {
+      margin:1rem 1rem 2rem 1rem;
+      background-color: white;
+      display: flex;
+      width: 28rem;
+      flex-direction: column;
+      box-shadow: 1px 2px 5px 2px;
```

Comment:
There is no color here, color is a required parameter for box-shadow.

Reply:
sure, will look into that